lib/active_merchant/billing/gateways/opp.rb

```diff
@@ -278,7 +295,9 @@ def add_options(post, options)
       end
 
       def build_url(url, authorization, options)
-        if options[:registrationId]
+        if options[:store]
+          "#{url.gsub(/payments/, 'registrations')}"
```

Comment:
Rubocop says:

```C: Style/UnneededInterpolation: Prefer to_s over string interpolation.```

since you're not doing any further modification to the result of `gsub`.

Reply:
@InPermutation thanks, fixed.